source/neuropod/backends/torchscript/torch_backend.cc

```diff
@@ -167,7 +167,7 @@ void insert_value_in_output(NeuropodValueMap & output,
 #endif
 
         // if tensor_type string or no tensor_type and empty list or list containing actual string
-        if ((has_type && tensor_type == TensorType::STRING_TENSOR) || (!has_type && list.size() == 0) ||
+        if ((has_type && tensor_type == TensorType::STRING_TENSOR) || (!has_type && list.empty()) ||
```

Comment:
Condition can be simplified to be more straightforward and exactly what comment says:

has_type && tensor_type == TensorType::STRING_TENSOR || !has_type && (list.empty() || list[0].isString())

Reply:
Probably not worth making any non `clang-tidy` related changes in this PR just to minimize the risk of introducing an issue (as this is a fairly large PR)